R/parcc_calcs.R

```diff
@@ -110,6 +110,9 @@ calculate_agg_parcc_prof <- function(end_year, subj, gradespan='3-11') {
     )
   )
   
+  all_grades <- bind_rows(all_grades, 
```

Comment:
I could go either way on this one.  On the one hand, it's kind of nice to have a method that returns back everything (including the layers of aggregation).  On the other hand, there are a lot of fun (meaning not so fun) geospatial dependencies that kick in once the geo layer is in play, and that might intimidate a new user.  I think I'm leaning toward leaving charter aggregates / all city aggregates / geo aggregates out of these base aggregation calls.

Reply:
sounds good -- took out this combination and added a bit more testing!